docs/infrastructure/madison-itb.md

```diff
@@ -0,0 +1,304 @@
+    1. Click the “Save” button
+    1. In the left navigation area, under “Actions”, click the “Sync” link
+
+1. Log in to `itb-host-1` and become `root`
+
+1. Create the libvirt definition file
+    1. Create a new XML file named after the desired hostname (e.g., `itb-ce2.xml`) and copy in the template below
+    1. Replace `{{ HOSTNAME }}` with the fully qualified hostname of the new virtual host
+    1. Replace `{{ MAC_ADDRESS }}` with the MAC address of the new virtual host (from above)
+    1. If desired, edit other values in the XML definition file; ask CHTC Infrastructure for help, if needed
+    1. Save the XML file
+    1. Create a new, empty disk image for the virtual host, in its correct location (as specified in the XML file):
+
+            :::console
+            truncate -s 192G /var/lib/libvirt/images/HOSTNAME.dd
+            chown qemu:qemu /var/lib/libvirt/images/HOSTNAME.dd
+
+    1. Load the new host definition into libvirt:
+
+            :::console
+            virsh define XML-FILE
+
+1. Install the new machine
+    1. Start the virtual machine:
+
+            :::console
+            virsh start HOSTNAME
+
+        At this time, the machine will boot over the network, and Cobbler will install and minimally configure the OS,
+        then reboot the now-installed machine.  The whole process typically takes 15–20 minutes.  You may be able to
+        `ssh` into the machine during the install process, but there is no need to monitor or interfere.
+
+    1. Once the machine is available (which you can only guess at), `ssh` in and verify that the machine basically works
+    1. Immediately run Ansible on the machine, first with the `secure.yml` playbook, then the `site.yml` one (see above)
+    1. Log in to the machine and look around to make sure it seems OK
+    1. When things look good, tell virsh to start the virtual machine when the host itself starts:
+
+            :::console
+            virsh autostart HOSTNAME
+
+### Libvirt VM Template
+
+``` xml
+<domain type='kvm'>
+
+  <name>{{ HOSTNAME }}</name>
+
+  <memory unit='GiB'>6</memory>
+  <vcpu>4</vcpu>
+  <os>
+    <type>hvm</type>
+    <boot dev='network'/>
+    <boot dev='hd'/>
+    <bios useserial='yes' rebootTimeout='0'/>
+  </os>
+  <features>
+    <acpi/>
+    <apic/>
+    <pae/>
+  </features>
+
+  <devices>
+
+    <emulator>/usr/libexec/qemu-kvm</emulator>
+
+    <disk type='file' device='disk'>
+      <source file='/var/lib/libvirt/images/{{ HOSTNAME }}.dd'/>
+      <target dev='vda' bus='virtio'/>
+    </disk>
+
+    <interface type='bridge'>
+      <mac address='{{ MAC_ADDRESS }}'/>
+      <source bridge='br0'/>
+      <model type='virtio'/>
+    </interface>
+
+    <serial type='pty'>
+      <target port='0'/>
+    </serial>
+    <console type='pty'>
+      <target type='serial' port='0'/>
+    </console>
+
+    <graphics type='vnc' autoport='yes' listen='127.0.0.1'/>
+
+  </devices>
+</domain>
+```
+
+## Archive
```

Comment:
Get rid of this section and find a different way to bug @matyasselmeci about cleaning up the "old stuff"